Makefile

```diff
@@ -0,0 +1,11 @@
+# create virtual environment
+.env:
+	virtualenv .env
+
+# install all needed for development
+develop: .env
+	.env/bin/pip install -e . -r requirements-testing.txt tox
+
+# clean the development envrironment
+clean:
+	-rm -rf .env
```

Comment:
Let's use `venv` instead. The `.env` is the default file used in foreman/honcho.